### DISCUSSION TITLE
Clarification of the Ne vs initial_size vs population_size parameters

### DISCUSSION THREAD

**Discussion by David-Peede** [tskit-dev/msprime#1816] [Q&A] [answered]
Hello All! I am working on converting `msprime` code written in 0.X to 1.X and I have seemed to confuse myself and would appreciate some clarification.  In `msprime` versions 0.Xs in the `simulate()` function you can pass the reference effective population size by using the `Ne` parameter, however, this is overwritten if you use `initial_size` parameter in the `PopulationConfiguration` function. In `msprime` 1.Xs the `population_size` parameter in the `sim_ancestry` function is analogous to the old `Ne` parameter with the exception that it cannot be used in tandem with the `initial_size` parameter in the `add_population` function. Is this interpretation correct? I apologize upfront if this is a simple question, but I have been doing mental gymnastics all morning and thought it would be best to seek clarification straight from the source!

Thanks in advance for any and all help!

-Dave

**Comment by jeromekelleher** [comment, marked as answer]
Hi @David-Peede :wave:

A lot of mental gymnastics has been done on this issue :wink:

Yes, you have it right. One thing we realised while doing the refactoring for msprime 1.0 is there is no "reference population size", it's just the sizes of the populations at various times. I had just confused myself into thinking there was.
